server/src/main/java/org/apache/kafka/server/config/KafkaLogConfigs.java

```diff
@@ -0,0 +1,170 @@
+            "If log.message.timestamp.type=CreateTime, the message will be rejected if the difference in timestamps exceeds " +
+            "this specified threshold. This configuration is ignored if log.message.timestamp.type=LogAppendTime.";
+
+    public static final String NUM_RECOVERY_THREADS_PER_DATA_DIR_CONFIG = "num.recovery.threads.per.data.dir";
+    public static final int NUM_RECOVERY_THREADS_PER_DATA_DIR_DEFAULT = 1;
+    public static final String NUM_RECOVERY_THREADS_PER_DATA_DIR_DOC = "The number of threads per data directory to be used for log recovery at startup and flushing at shutdown";
+
+    public static final String AUTO_CREATE_TOPICS_ENABLE_CONFIG = "auto.create.topics.enable";
+    public static final boolean AUTO_CREATE_TOPICS_ENABLE_DEFAULT = true;
+    public static final String AUTO_CREATE_TOPICS_ENABLE_DOC = "Enable auto creation of topic on the server.";
+
+    public static final String MIN_IN_SYNC_REPLICAS_CONFIG = ServerTopicConfigSynonyms.serverSynonym(TopicConfig.MIN_IN_SYNC_REPLICAS_CONFIG);
+    public static final String MIN_IN_SYNC_REPLICAS_DOC = "When a producer sets acks to \"all\" (or \"-1\"), " +
+            "<code>min.insync.replicas</code> specifies the minimum number of replicas that must acknowledge " +
+            "a write for the write to be considered successful. If this minimum cannot be met, " +
+            "then the producer will raise an exception (either <code>NotEnoughReplicas</code> or " +
+            "<code>NotEnoughReplicasAfterAppend</code>).<br>When used together, <code>min.insync.replicas</code> and acks " +
+            "allow you to enforce greater durability guarantees. A typical scenario would be to " +
+            "create a topic with a replication factor of 3, set <code>min.insync.replicas</code> to 2, and " +
+            "produce with acks of \"all\". This will ensure that the producer raises an exception " +
+            "if a majority of replicas do not receive a write.";
+
+    public static final String CREATE_TOPIC_POLICY_CLASS_NAME_CONFIG = "create.topic.policy.class.name";
+    public static final String CREATE_TOPIC_POLICY_CLASS_NAME_DOC = "The create topic policy class that should be used for validation. The class should " +
+            "implement the <code>org.apache.kafka.server.policy.CreateTopicPolicy</code> interface.";
+    public static final String ALTER_CONFIG_POLICY_CLASS_NAME_CONFIG = "alter.config.policy.class.name";
+    public static final String ALTER_CONFIG_POLICY_CLASS_NAME_DOC = "The alter configs policy class that should be used for validation. The class should " +
+            "implement the <code>org.apache.kafka.server.policy.AlterConfigPolicy</code> interface.";
+    public static final String LOG_MESSAGE_DOWNCONVERSION_ENABLE_CONFIG = ServerTopicConfigSynonyms.serverSynonym(TopicConfig.MESSAGE_DOWNCONVERSION_ENABLE_CONFIG);
```

Comment:
Nit: There's a mix of `DOWNCONVERSION` and `DOWN_CONVERSION`.

Reply:
Good spot. fixed now